src/oss/confluent.sh

```diff
@@ -620,7 +625,18 @@ service_exists() {
 
 command_exists() {
     local command="${1}"
-    exists "${command}" "commands"
+    exists "${command}" "commands" || exists "${command}" "enterprise_commands"
+}
+
+cli_file_exists(){
```

Comment:
I don't see much value to this function in its current form. Can be re-written inline (see comment inline below)